### ISSUE TITLE
Migrate tests from `@actions` functions to `@actions` API functions

### ISSUE THREAD

**Issue by NicoSerranoP** [privacy-scaling-explorations/p0tion#325]
# Motivation
We are moving all the p0tion coordinator infrastructure to a single server application (API). We replicated some of the functions from `@actions` to be compatible with this new API. These functions have the same name plus a suffix `API. For example:

`isCoordinator`
`isCoordinatorAPI`

We need to write tests for all these new `API` functions. We can use the existing tests as a guide but make sure to change anything to make it fit.

**Comment by 0xgun** [comment]
Can I work on this ?

**Comment by NicoSerranoP** [comment]
@0xgun Yes sure! I just updated the issue description with a tutorial to setup The API coordinator in case you need it :)

**Comment by 0xgun** [comment]
I'm not able to find the `/packages/api` directory.

**Comment by NicoSerranoP** [comment]
Sorry it is in the `feat/nico/cfs-in-backend` branch
